packages/jsii/lib/assembler.ts

```diff
@@ -30,6 +30,9 @@ export class Assembler implements Emitter {
   private _deferred = new Array<DeferredRecord>();
   private _types: { [fqn: string]: spec.Type } = {};
 
+  /** Map of Symbol to namespace export Symbol */
+  private readonly _namespaceMap = new Map<ts.Symbol, ts.Symbol>();
```

Comment:
Maybe rename to something like `submoduleMap` so it has semantic connection to submodules